src/Chapter14/Listing14.08.UsingThePlusAndMinusDelegateOperators.cs

```diff
@@ -22,12 +24,19 @@ public static void Main()
             delegate2 = cooler.OnTemperatureChanged;
 
             Console.WriteLine("Combine delegates using + operator:");
+            #region HIGHLIGHT
             delegate3 = delegate1 + delegate2;
+            #endregion HIGHLIGHT
             delegate3(60);
 
             Console.WriteLine("Uncombine delegates using - operator:");
+            #region HIGHLIGHT
             delegate3 = (delegate3 - delegate2)!;
+            #endregion HIGHLIGHT
             delegate3(60);
+            #region EXCLUDE
         }
     }
+    #endregion EXCLUDE
+    #endregion INCLUDE
 }
```

Comment:
Would it be simpler to just move the #region INCLUDE to just surround the inner section instead of the #exclude right next to an include?